code/src/test/java/com/expediagroup/sdk/core/http/RequestBodyJavaTest.java

```diff
@@ -0,0 +1,72 @@
+package com.expediagroup.sdk.core.http;
+
+import okio.Buffer;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.util.HashMap;
+
+public class RequestBodyJavaTest {
+
+    @Test
+    @DisplayName("should access RequestBody.create as static method with InputStream only")
+    public void shouldCreateRequestBodyFromInputStreamOnly() {
+        String content = "Hello World";
+        InputStream inputStream = new ByteArrayInputStream(content.getBytes());
+        RequestBody.create(inputStream);
+    }
+
+    @Test
+    @DisplayName("should access RequestBody.create as static method with InputStream, media type, content length")
+    public void shouldCreateRequestBodyFromInputStreamAndMediaTypeAndContentLength() {
+        String content = "Hello World";
+        InputStream inputStream = new ByteArrayInputStream(content.getBytes());
+        RequestBody.create(inputStream, MediaType.Companion.getTEXT_PLAIN(), content.getBytes().length);
```

Comment:
Can you please annotate the properties in the MediaType companion object with `@JvmField` so we can call them like `MediaType.TEXT_PLAIN` from Java?

Reply:
This was addressed in PR #117 
Will fix everything related to `MediaType` once we merge it